transport/swift/swift_test.go

```diff
@@ -0,0 +1,142 @@
+package swift
+
+import (
+	"context"
+	"github.com/imgproxy/imgproxy/v3/config"
+	"github.com/ncw/swift/v2"
+	"github.com/ncw/swift/v2/swifttest"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+const (
+	testContainer = "test"
+	testObject    = "test.png"
+)
+
+func initTestServer() (*swifttest.SwiftServer, error) {
+	server, err := swifttest.NewSwiftServer("localhost")
+
+	config.SwiftAuthURL = server.AuthURL
+	config.SwiftUsername = swifttest.TEST_ACCOUNT
+	config.SwiftAPIKey = swifttest.TEST_ACCOUNT
+	config.SwiftAuthVersion = 1
+
+	return server, err
+}
+
+func setupTestFile(t *testing.T) {
```

Comment:
Please rewrite tests to use `suite.Suite` from testify. This way you can initialize the server and upload the file once before all the tests.